k8s/base/root-ingress.yaml

```diff
@@ -3,7 +3,6 @@ kind: Ingress
 metadata:
   name: root-ingress
   annotations:
-    nginx.ingress.kubernetes.io/ssl-redirect: "false"
     nginx.ingress.kubernetes.io/app-root: /gridexplore
```

Comment:
will have 
`nginx.ingress.kubernetes.io/service-upstream: "true"`
from ingresses.yml patch
it did'nt have it before no ? is it ok ?

Reply:
yes for this one I chose to make a new conf just to keep things simple, because it would mean to create something just to remove it when I make thing more standardized